### ISSUE TITLE
assignment #2 feedback

### ISSUE THREAD

**Issue by nsonnad** [lucypapachristou/chart-assignment#1]
Hey Lucy! This is a very good and important piece; my apologies if my delay has gotten in the way of your publishing it, let me know if you want to discuss that further.

I'll just get a couple chart-related things out of the way before going more into detail about the piece. I think your chart hed and dek are a little verbose and redundant and can be cut down to the bare minimum of what we should be looking at. People can infer the numbers and dates, you just need to tell them the high-level takeaway: "This year Austria overtook the rest of Europe in asylum applications from unaccompanied minors" or something to that effect. Then you don't really need a dek, but I would use that dek space to label your units ("# of asylum applications made by unaccompanied minors" or whatever). Lastly on the chart, I might suggest making all the other countries a medium gray or something muted to draw attention to the Austria line and make it easier to follow it.

Moving on to the larger piece. I don't mind the length, but I think you can spend less time in certain places, and I think some structural changes would make it easier to follow.

So here's what I'll say: I think it's certainly necessary to attempt to explain the Austria increase, but I feel that pursuing several of these different pathways of explanation made me lose the main thread of the story. And since they are all pretty speculative, I'd probably just roll all of them into a sentence or two after "The numbers are stark, but Austria asylum experts don’t have a clear understanding of why..." After that, you could quickly run through some of the speculations and add that we don't really know.

Then I'd move right on to the idea of this being a crisis of management, skipping the forays into the Bulgarian mafia, the instability of Afghanistan, etc. It seems like probably all these things are factors to some extent. So I'd like to see you just bring up this very clear line and use that to start the next discussion: "Such a rapid increase in the amount of children entering Austria this year created a backlog in the asylum system. But it was a backlog that was entirely avoidable, experts concur." Then introduce Glawischnig, the comparisons with 2015, and the managerial failures that have led to this, the länder etc.

Finally, you can end with the difficult conditions these kids face and transition out of that back to Mohammed.

To summarize, I'm advocating a couple things here. One is cutting a lot of the speculation, the other is keeping the sections tightly focused on one aspect of the story before moving on to the next. In the latter two-thirds I think you move back and forth a bit between the challenges for kids, the managerial failures, and the possible reasons. You might even put a couple section headers (`<h2>`!) to signal both to yourself and to readers what to focus on at a given time.

Does that make sense? I hope so! Feel free to slack me if you want to discuss further, but overall great work and I don't see any reason why you can't publish this.

**Comment by lucypapachristou** [comment]
Hi Nikhil,

Thanks for the feedback, super helpful. I take your points on the chart,
and I'll make the changes!

As for the piece itself: you're right, it needs severe restructuring. I
didn't have much time after I wrote it to really edit it before the
deadline, but I'm still working on it and doing a little more reporting
too. My aim is to cut it in half basically, so yes, tighten everything.

In all, I really enjoyed reporting this and enjoyed learning how to make
the chart, because in the case of this story the visual of the chart really
adds to the story/helps tell the story. I'll let you know if it makes its
way out into the world!

Best,
Lucy
